Sources/PointFree/Episode/Episode.swift

```diff
@@ -63,6 +78,17 @@ public struct Episode {
     }
   }
 
+  public var freeSince: Date? {
```

Comment:
I want to use this to potentially bump up old episodes that have recently be switched to free. it's not currently working (I have it trello'd), but that's why this is here